.github/workflows/ci.yml

```diff
@@ -33,20 +33,25 @@ jobs:
       data-aggregator-tag: ${{ steps.setup_tags.outputs.data-aggregator-tag}}
       gateway-api-tag: ${{ steps.setup_tags.outputs.gateway-api-tag}}
       database-migrations-tag: ${{ steps.setup_tags.outputs.database-migrations-tag}}
-
+    permissions:
+      id-token: write
+      contents: read
+      pull-requests: read
     steps:
       - name: Dump context
         uses: crazy-max/ghaction-dump-context@516dbb0c760f39b4cdd750ae095f1688780f68f4
       - uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b
         with:
           fetch-depth: 0
 
-      - name: Login to GCR
-        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
+      - id: auth
+        uses: google-github-actions/auth@v0.4.0
```

Comment:
Aren't we supposed to use commit SHA rather version number (`v0.4.0`)?

Reply:
Yes, changed ✔️